Oxide.Core/Libraries/WebRequests.cs

```diff
@@ -100,6 +134,17 @@ public WebRequest(string url, Action<int, string> callback, Plugin owner)
                 removedFromManager = Owner?.OnRemovedFromManager.Add(owner_OnRemovedFromManager);
             }
 
+            /// <summary>
+            /// Initializes a new instance of the WebRequest class
+            /// </summary>
+            /// <param name="url"></param>
+            /// <param name="owner"></param>
+            /// <param name="callback"></param>
+            public WebRequest(string url, Plugin owner, Action<WebResponse> callback) : this(url, null, owner)
+            {
+                CallbackV2 = callback;
```

Comment:
name `CallbackV2` looks very out of pattern.

Reply:
Can't really call it the same thing.